pkgs/games/arx-libertatis/default.nix

```diff
@@ -38,11 +37,14 @@ stdenv.mkDerivation rec {
   ];
 
   enableParallelBuilding = true;
+  dontWrapQtApps = true;
 
   postInstall = ''
     ln -sf \
       ${dejavu_fonts}/share/fonts/truetype/DejaVuSansMono.ttf \
       $out/share/games/arx/misc/dejavusansmono.ttf
+  '' + optionalString withCrashReporter ''
+    wrapQtApp "$out/libexec/arxcrashreporter"
```

Comment:
Binaries in `libexec` should be automatically wrapped.
Is this a script?

Reply:
No, but I had to disable the wrappeing by default since it was acting on other unrelated binaries: only the crash reporter is a Qt application.